packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1088,6 +1088,18 @@ base class PipelineOwner with DiagnosticableTreeMixin {
   bool _shouldMergeDirtyNodes = false;
   List<RenderObject> _nodesNeedingLayout = <RenderObject>[];
 
+  /// The [RenderObject]s which need to be laid out in the next [flushLayout] pass.
+  ///
+  /// [RenderObject]s with [RenderObject.isRepaintBoundary] are added
+  /// when they are marked for layout. Subclasses of [PipelineOwner] may use them
+  /// to invalidate caches or otherwise make performance optimizations.
+  /// Since nodes may be marked for layout at any time, they are best checked during
+  /// [flushLayout].
+  ///
+  /// Note that this does not include marked children of child [PipelineOwner]s.
+  @protected
```

Comment:
`@nonVirtual` too, if you're not overriding it?

Reply:
Done